rspec-puppet-facts.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'voxpupuli-rubocop', '~> 2.0'
 
+  s.add_runtime_dependency 'deep_merge', '~> 1.2'
```

Comment:
in Vox Pupuli, we would do `'~> 1.2', '>= 1.2.2'` for new deps to ensure that we don't pull in a major version but also the latest version that we used for testing.

Reply:
I'm not a fan of such strict requirements. They should be as open as possible.

Reply:
Open to either, just let me know which you'd prefer!

Reply:
when in doubt, always choose the opinion from @ekohl, not mine :)